salt/prometheus/files/conf-prometheus.yml

```diff
@@ -11,11 +11,11 @@ scrape_configs:
 
   # Node Exporter.
 {%- for job_name, host in {
-    'ocds-cove-live.oc4ids': 'ocp01.open-contracting.org',
+    'ocds-cove-live.oc4ids': 'ocp17.open-contracting.org',
```

Comment:
@jpmckinney , please could you suggest a better `job_name` for prometheus?
I would rather the instances reported in as new entities rather than replacing the previous servers - happy to be overruled tho.

Reply:
Sure, maybe we just use the same name as in the Salt roster, and increment with each time that service has been redeployed? So it would now be `cove-oc4ids2` (if we pretend the original was `cove-oc4ids1` instead of `ocds-cove-live.oc4ids`).

Reply:
Then for the other one: cove-ocds-2 (I don't know where ocds-live3 comes from, and I don't care to find out ;)).